examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -71,12 +71,18 @@ const containerSchema = {
 function createDiceRollerControllerProps(map: ISharedMap): DiceRollerControllerProps {
 	return {
 		get: (key: string) => map.get(key) as number,
-		set: (key: string, value: any) => map.set(key, value),
-		on(event: "valueChanged", listener: (args: IValueChanged) => void) {
+		set: (key: string, value: unknown) => map.set(key, value),
```

Comment:
Just a question for my own understanding; why is unknown preferred over any?

Reply:
`any` allows users to call any properties or methods on the object, and the compiler assumes they all are valid and exist at runtime. Additionally, `any` is assignable to arguments of any type without validation. So, it promotes very unsafe code.

`unknown` doesn't allow you to assume anything about the object - no methods or properties may be called on an object of this type, and `unknown` is not assignable to any types except `unknown` and `any`. So, using `unknown` forces you to do validation before making any assumptions about what may or may not be there.